### PR TITLE
Update chat

### PR DESCRIPTION
**Pull request by Quentin-Macquart**

### CODE REVIEW

File: src/pages/Supplies/index.jsx [hunk 1]
```diff
@@ -4,40 +4,75 @@ import fournitures from 'assets/Img/fournitures.png';
...
-                </div>
-              </div>
-            );
-          })}
-        </div>
-        <section>
-          <MainButton className="addToCart" content="Ajouter au panier" />
-        </section>
+        {console.log(modal)}
+        <Header logo={fournitures} />{' '}
```
Comment (HeloiseFerla) [inline src/pages/Supplies/index.jsx]: ?

File: src/pages/Supplies/index.jsx [hunk 2]
```diff
@@ -4,40 +4,75 @@ import fournitures from 'assets/Img/fournitures.png';
...
+                );
+              })}
+            </div>
+            <MainButton
+              className="watchCart"
+              content="Voir mon panier"
+              clickCallback={() => {
+                toggleModal();
+              }}
+            />{' '}
```
Comment (HeloiseFerla) [inline src/pages/Supplies/index.jsx]: ?

File: src/pages/SuppliesModal/index.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  const sendData = () => {
+    const myOrder = cart
+      .filter((item) => item > 0)
+      .map((item) => {
+        return {
+          itemId: supplyElement[item].id,
+          qtty: item,
+        };
+      });
+
```
Comment (HeloiseFerla) [inline src/pages/SuppliesModal/index.jsx]: empty line

File: src/pages/SuppliesModal/style.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  height: 100vh;
+  z-index: 10;
+
+  h2 {
+    display: flex;
+    justify-content: center;
+    font-weight: bold;
+    padding-top: 20px;
+  }
+
```
Comment (HeloiseFerla) [inline src/pages/SuppliesModal/style.jsx]: empty line

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/SuppliesModal/index.jsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
+// import { useState } from 'react';
```
Comment (Hacene010) [inline src/pages/SuppliesModal/index.jsx]: dead code

File: src/pages/SuppliesModal/index.jsx [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+          qtty: item,
+        };
+      });
+    axios
+      .post(
+        `${process.env.REACT_APP_API_URL}/supplies/${user.id}/cartSupplies`,
+        myOrder
+      )
+      .then((data) => {
+        console.log(data);
```
Comment (Hacene010) [inline src/pages/SuppliesModal/index.jsx]: ?

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Chat/index.jsx [hunk 7]
```diff
@@ -49,26 +53,37 @@ export default function Chat() {
...
-              className = 'notice';
+          {messages.map(
+            ({ id, userId, content, time, picture, firstname, lastname }) => {
+              let className = 'alienMsg';
+              if (userId !== user.id) {
+                className = 'notice';
+              }
+              if (userId === user.id) {
+                className = 'ownMsg';
+              }
```
Comment (LoicBrassart) [inline src/pages/Chat/index.jsx]: `else` ?

File: src/pages/SuppliesModal/index.jsx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+        };
+      });
+    axios
+      .post(
+        `${process.env.REACT_APP_API_URL}/supplies/${user.id}/cartSupplies`,
+        myOrder
+      )
+      .catch((e) => {
+        console.log(e);
+      });
```
Comment (LoicBrassart) [inline src/pages/SuppliesModal/index.jsx]: No console.log

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Several branches not created from dev -> I read several times the same code -> I'm less nice in my reviews :p

**Review by Hacene010 — DISMISSED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]

**Pull request merged**